README.md

```diff
@@ -119,48 +127,50 @@ db = MySQLDatabase(
     **connect_kwargs,
 )
 
-class TestModel(Model):
-    class Meta:
-        database = db
-        table_name = 'test'
-
+class DocumentModel(Model):
     embedding = VectorField(3)
-
-# or add hnsw index when creating table
-class TestModelWithIndex(Model):
     class Meta:
         database = db
-        table_name = 'test_with_index'
-
-    embedding = VectorField(3, constraints=[SQL("COMMENT 'hnsw(distance=l2)'")])
-
+        table_name = 'peewee_documents'
 
 db.connect()
-db.create_tables([TestModel, TestModelWithIndex])
+db.create_tables([DocumentModel])
 ```
 
 Insert vector data
 
 ```python
-TestModel.create(embedding=[1, 2, 3])
+DocumentModel.create(embedding=[1, 2, 3])
 ```
 
 Get the nearest neighbors
 
 ```python
-TestModel.select().order_by(TestModel.embedding.l2_distance([1, 2, 3.1])).limit(5)
+DocumentModel.select().order_by(DocumentModel.embedding.l2_distance([1, 2, 3.1])).limit(5)
 ```
 
 Get the distance
 
 ```python
-TestModel.select(TestModel.embedding.cosine_distance([1, 2, 3.1]).alias('distance'))
+DocumentModel.select(DocumentModel.embedding.cosine_distance([1, 2, 3.1]).alias('distance'))
 ```
 
 Get within a certain distance
 
 ```python
-TestModel.select().where(TestModel.embedding.l2_distance([1, 2, 3.1]) < 0.5)
+DocumentModel.select().where(DocumentModel.embedding.l2_distance([1, 2, 3.1]) < 0.5)
+```
+
+Add hnsw index
+
+```python
+# vector index currently depends on tiflash
```

Comment:
This is by design and permanent. If TiFlash component is not deployed corresponding operation should fail, and asking user to deploy a TiFlash in order to use Vector Index (just like FULL TEXT INDEX will fail in MySQL if ENGINE is MEMORY).